### ISSUE TITLE
Issue with comms.py

### ISSUE THREAD

**Issue by ThibltJ** [mikeshultz/ledger-eth-lib#27]
When I want to retrieve the accounts from the ledger (v0.7.1), I have this error : 
```bash
File "../lib/python3.8/site-packages/ledgereth/comms.py", line 168, in init_dongle
    if not is_usable_version(DONGLE_CONFIG_CACHE):
NameError: name 'DONGLE_CONFIG_CACHE' is not defined
```
The workaround that I found is the initialisation of the global variables 

```python
DONGLE_CACHE: Dongle = None
DONGLE_CONFIG_CACHE: bytes = None
```
Let me know if I can give you more information in order to help you

**Comment by mikeshultz** [comment]
Thanks for the report.  Looking into it.

**Comment by mikeshultz** [comment]
Fix went out with v0.7.2.
